awx/main/migrations/0177_change_cyberark_plugin_names.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 3.2.16 on 2023-03-14 18:25
+
+from django.db import migrations
+
+from awx.main.models import CredentialType
+
+
+class Migration(migrations.Migration):
+    def update_cyberark_ccp_name(apps, schema_editor):
+        CredentialType.setup_tower_managed_defaults(apps)
+        aim_types = apps.get_model('main', 'CredentialType').objects.filter(namespace='aim').order_by('id')
+
+        if aim_types.count() == 2:
+            original, renamed = aim_types.all()
```

Comment:
I don't think this ordering is guaranteed.

Reply:
Fixed. Order now by 'changed'.